components/brave_wallet_ui/stories/wallet-extension-panels.tsx

```diff
@@ -396,7 +402,9 @@ export const _ConnectedPanel = (args: { locked: boolean }) => {
       id: '1',
       name: 'Account 1',
       address: '1',
-      balance: '0.31178',
+      nativeBalanceRegistry: {
+        ETH: '0.31178'
```

Comment:
How do we determine if this is wei oe eth?

Reply:
Not sure why this mock data has this value, probably is good to investigate.
Please reference the `diff` 😎  relevancy in this PR is just updating this object to the changed `type`.
Best to keep unrelated things separate.